src/board/v3/nap/nap_common.c

```diff
@@ -26,8 +26,7 @@
 
 #define PROCESS_PERIOD_ms (1000)
 
-void nap_isr(void *context);
-void nap_unlock(const u8 key[]);
```

Comment:
You might need this forward declaration if you want to unlock the NAP in `nap_setup()`